lib/utils/normalization.ts

```diff
@@ -21,6 +21,7 @@ export function normalizeUuidProp(
 	propName = 'uuid',
 ) {
 	if (_.isNumber(params[propName])) {
-		params[propName] = _.toString(params[propName]);
+		params[propName] =
+			params[propName + '_raw'] || _.toString(params[propName]);
```

Comment:
This is the change that matters in this PR. It builds on this other Capitano PR:
https://github.com/balena-io/capitano/pull/70/files

Reply:
LOL, so Capitano converts it from a string to a number, and balena-cli turns it back from a number to a string :slightly_smiling_face:   (but I agree that given the complexity, it's probably worth doing the minimum change possible!)